src/nix/daemon.cc

```diff
@@ -500,6 +500,45 @@ static RegisterLegacyCommand r_nix_daemon("nix-daemon", main_nix_daemon);
 
 struct CmdDaemon : StoreCommand
 {
+    bool stdio = false;
+    std::optional<TrustedFlag> isTrustedOpt = std::nullopt;
+
+    CmdDaemon()
+    {
+        addFlag({
+            .longName = "stdio",
+            .description = "Attach to standard I/O, instead of trying to bind to a UNIX socket.",
+            .handler = {&stdio, true},
+        });
+
+        addFlag({
+            .longName = "force-trusted",
+            .description = "Forces the daemon to trust connecting clients, forwarding the connection without the receiving daemon processing it.",
```

Comment:
Does this actually forward the connection? I assume that depends on `nix daemon`'s own `--store` parameter: If it's "local", then it won't forward to another daemon.

Reply:
That is true. Forcing untrusted does make not force in the `--stdio` case when `--store` is a `RemoteStore` subclass, but forcing trusted (if I am reading the code above right) doesn't seem to affect when forwarding happens.

Reply:
Yes I think that is good